templates/impala/base.html

```diff
@@ -45,7 +45,7 @@
 
     {{ js('preload') }}
   </head>
-  <body class="html-{{ DIR }} {{ request.APP.short }} moz-header-slim {% block bodyclass %}{% endblock %} is-impala"
+  <body class="html-{{ DIR }} {{ request.APP.short }}  moz-header-slim {% block bodyclass %}{% endblock %} is-impala {% if waffle.flag('restyle') %}restyle{% endif %}"
```

Comment:
I don't think the leading space is needed in front of `moz-header-slim`.

Also the space for the `restyle` class could be inside the if block so there's not an extra space when it's not present.